docker/build-tools/Dockerfile

```diff
@@ -48,15 +48,14 @@ ENV GCR_AUTH_VERSION=2.1.21
 ENV GH_VERSION=2.55.0
 ENV GOCOVMERGE_VERSION=b5bfa59ec0adc420475f97f89b58045c721d761c
 ENV GOIMPORTS_VERSION=v0.24.0
-# When updating the golangci version, you may want to update the common-files config/.golangci* files as well.
```

Comment:
We should probably keep this

Reply:
The file removed the version we pinned to. Now it just uses the version we built in.

Reply:
While the version pinning got removed, there are times where rules need to be changed between versions, so it's a good reminder to ensure the rules are still correct before bumping and causing org-wide lint test failures.